ibis/backends/tests/test_array.py

```diff
@@ -78,7 +77,12 @@ def test_array_scalar(con, backend):
         assert con.execute(expr.typeof()) == ARRAY_BACKEND_TYPES[backend_name]
 
 
-@pytest.mark.notimpl(["polars", "flink"], raises=com.OperationNotDefinedError)
+@pytest.mark.notimpl(["polars"], raises=com.OperationNotDefinedError)
+@pytest.mark.notimpl(
+    ["flink"],
+    raises=com.OperationNotDefinedError,
+    reason="No translation rule for <class 'ibis.expr.operations.arrays.ArrayRepeat'>",
```

Comment:
If the raised exception type is `com.OperationNotDefinedError` you don't need to include the reason, especially if the reason is just a restatement of the error message.

Reply:
Sure, removed the reasons for `com.OperationNotDefinedError`'s.